### DISCUSSION TITLE
Difference between dawn & day buffers

### DISCUSSION THREAD

**Discussion by stevie445** [nathanmarlor/foxess_em#250] [General]
Hi.
Just like to start by saying, what a fantastic job you've made on this!  Love it, well done!

Simple question, what's different about the 2 buffer settings?  I couldn't see a description for the function of each

I would assume both allow a off-leak stored charge prior to the start of the day, so why do we need two? e.g. is the day buffer affected by Solcast and the other isn't or......(???). Just not sure & sorry if this has been asked before

**Comment by stevie445** [comment]
Just to add, it would be useful if the day buffer reduced if the solar forecast is over the capacity of the batteries.  Just a thought

**Comment by nathanmarlor** [comment]
Thanks @stevie445 - the buffers are there to avoid the batteries finishing on 0% (usable) capacity as house load, forecasts etc. can change against predictions. 

Dawn load is calculated as usage between the end of off-peak and solar being greater than house load.
Day load is calculated as usage between peak and off-peak. 

I usually set dawn buffer to 1kwh so at the time the solar takes over from the batteries I have ~20% capacity remaining. I have my day buffer set to 2kwh so at the end of the day I have ~30% remaining. 

Hopefully that makes sense?

**Comment by stevie445** [comment]
Ah.. perfect.  Thanks!

**Comment by stevie445** [comment]
I defo have more of a problem in the morning so I'll adjust up.  Any thoughts on the ignore the day buffer if solar forecast is above battery capacity?

Again, great job.  I'm just OCD and like to squeeze everything :)

**Reply by stevie445** [reply]
Sorry, I see you addressed that in your first line
